x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -241,6 +241,11 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables the Defend Insights feature
    */
   defendInsights: false,
+
+  /**
+   * Enables flyout history
+   */
+  flyoutHistoryEnabled: false,
```

Comment:
@PhilippeOberti 
I'm debating whether to change the flag to something like `expandableFlyoutNewNav` or `flyoutNewPreviewNav`, so that it can be reused with the navigation PR, they kind of go hand in hand, wdyt?

Reply:
I think what you have now is ok, and there might be value in having 2 feature flag... But merging them is not a bad idea either. Why about just `expandableFlyoutNavigation`?

Reply:
Updated to `newExpandableFlyoutNavigationEnabled` (added `new` since there is navigation in flyout right now), also added placeholder empty state in a876cbee9392f91875be96ceaaa8124b11af19b0